src/codegen/cfg.rs

```diff
@@ -63,7 +63,7 @@ pub enum Instr {
     /// Abort execution
     AssertFailure { encoded_args: Option<Expression> },
     /// Print to log message
-    Print { expr: Expression },
+    Print { expr: Expression, is_error: bool },
```

Comment:
It is not clear to me what `is_error` does. It should either have a comment explaining it or be removed.

Reply:
is_error is used in the print function to decide on whether we should log seal_debug_message or not. I though it would make sense to mask the user from logging a seal_debug_message after a runtime error has been reported.

Reply:
@xermicus What do you think?

Reply:
I vouch for removal. A print instruction can always behave the same, no need to hide it from the user IMO. It only makes the code more complicated